setup.py

```diff
@@ -29,8 +32,20 @@ def get_version(*file_paths):
     author_email="contact@edunext.co",
     url="https://github.com/eduNEXT/eox-core",
     description="eduNEXT Openedx extensions",
-    long_description="",
-    install_requires=[],
+    long_description=README,
+    long_description_content_type='text/x-rst',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Framework :: Django :: 1.11',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: GNU Affero General Public License v3',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+    ],
+    install_requires=["celery", "django", "edx-opaque-keys[django]", "edx-django-oauth2-provider", "djangorestframework"],
```

Comment:
we should have this with oep18

Reply:
Split each of the packages on their own line. @morenol